wiki/Gameplay/Daily_challenge/en.md

```diff
@@ -0,0 +1,61 @@
+
+Every beatmap comes from the [Featured Artist](/wiki/People/Featured_Artists) listing, and sometimes the maps come with a mod forced onto users, which requires them to pass the map with the selected mod.
+
+## Game menu
+
+From the main menu, the daily challenge menu can be accessed with the following steps:
+
+1. Click the `Play` button or press `P`.
+2. Click the `Daily challenge` button or press `D`.
+
+![](img/daily-challenge-menu.png "Screenshot of the daily challenge menu during November 7, 2024.") 
+
+On entering, the user is presented with an intro showing the beatmap the user has to pass, and the mods they have to pass it with.
+
+After the intro ends, the user can view the data for today's challenge. The left side displays various score-related information like the total pass count and the cumulative total score. In the middle, a leaderboard shows the top scores achieved by players. Users are able to discuss the daily challenge in the chat on the right.
+
+## Beatmap difficulty range
+
+![](img/Beatmap-selection-criteria.png "Specific star ratings beatmaps have to follow for the entire week")
+
+## Streak milestones
+
+Play streaks are displayed on the user profile with different colours depending on certain streak milestones.
+
+### Rank breakdown 
+
+| Rank | Total streak | Daily streak | Weekly streak |
+| :-: | :-: | :-: | :-: |
+| Lustrous | 1081 days | 360 days | 53 weeks |
+| Radiant | 721 days | 240 days | 36 weeks |
+| Rhodium | 361 days | 120 days | 19 weeks |
+| Platinum | 181 days | 60 days | 10 weeks |
+| Gold | 91 days | 30 days | 6 weeks |
+| Silver | 31 days | 10 days | 3 weeks |
+| Bronze | 16 days | 5 days | 2 weeks |
+| Iron | less than 16 days | less than 5 days | less than 2 weeks |
+
+## Contributors
+
+| Name | Role |
+| :-- | :-- |
+| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | Organisation effort lead |
+| ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Beatmap picking |
+| ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9501251) | Beatmap picking |
+| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | Beatmap picking |
+| ::{ flag=GB }:: [DeviousPanda](https://osu.ppy.sh/users/4966334) | Beatmap picking |
+| ::{ flag=GB }:: [kingling](https://osu.ppy.sh/users/7010761) | Beatmap picking |
+| ::{ flag=KR }:: [Luscent](https://osu.ppy.sh/users/2688581) | Beatmap picking |
+
+## Trivia
+
+::: Infobox
+![](img/yt-comment.jpg "waxxx14's YouTube comment suggesting the \"cup of the day\" format")
+:::
+
+- The idea for daily challenge was taken from a comment from waxxx14 requesting a "cup of the day" format from [TrackMania](https://en.wikipedia.org/wiki/TrackMania) in osu!, under a lazer dev video, ["deciding what to do with lazer"](https://www.youtube.com/watch?v=xUSxEjQQ1UI)
```

Comment:
i think development makes more sense since even tho it was an update video, it also served a purpose to get community feedback on what should be the next main focus for laser + it just sounds nicer IMO.

Reply:
Yeah sure :thumbs_up: